Sizes/Classes/UI/SizesWindow.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SizesWindow.swift
+//  Pods-Sizes_Example
+//
+//  Created by Bahadir Oncel on 14.10.2018.
+//
+
+import UIKit
+
+open class SizesWindow: UIWindow {
+    
+    private(set) open var containedRootViewController: UIViewController?
+    private(set) open var sizesViewController: SizesViewController?
+    
+    override open var rootViewController: UIViewController? {
```

Comment:
I think we can do something along this lines to avoid having a `containedRootViewController` property and make it a bit simpler: 

```swift
open class SizesWindow: UIWindow {
    
    public let sizesViewController = SizesViewController()

    override open var rootViewController: UIViewController? {
        get {
            return sizesViewController.containedController
        }
        set {
            guard let root = newValue, !root.isKind(of: SizesViewController.self) else {
                return
            }
            super.rootViewController = sizesViewController
            sizesViewController.contain(viewController: root)
        }
    }
}
```

Reply:
I didn't know I could access `super.rootViewController` while also overriding it in a subclass. This solution worked and I agree that it's simpler & better.